src/components/role.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import type { GameRole } from '$lib/stores/game';
+	import { t } from '$lib/stores/i18n';
+
+	export let role: GameRole;
+
+	export let disabled = false;
+	export let listView = false;
+
+	$: name = $t(`narrator.${role.name}.name`);
+</script>
+
+<button class="avatar indicator" on:click {disabled}>
+	{#if !listView && role.amount > 1}
```

Comment:
I would prefer roles having the batch as soon as they are selected. IMHO it would be a bit clearer which roles are selected when you use the app for the first time.

Reply:
mhm. isn't that kinda weird because there can be only one e.g. amor or whitch? Maybe instead we can implement a tutorial mode, which will guide first time users with tooltips?

Reply:
Thats a good idea! Maybe we just add a heading 'Selected Roles' or 'Active Roles' and keep the badges as they are?

Reply:
👍 say no more

Reply:
done

Reply:
https://github.com/BjarneRentz/werwapp/issues/46